src/test/java/com/amazon/dlic/auth/ldap/LdapBackendTest.java

```diff
@@ -530,7 +531,7 @@ public void testLdapAuthorizationOnly() throws Exception {
         Assert.assertNotNull(user);
         Assert.assertEquals("jacksonm", user.getName());
         Assert.assertEquals(2, user.getRoles().size());
-        Assert.assertEquals("ceo", new ArrayList(new TreeSet(user.getRoles())).get(0));
+        MatcherAssert.assertThat(user.getRoles(), hasItem("ceo"));
```

Comment:
we should replace this with static import, all instances

Reply:
This is a good candidate for a checkstyle / spotless rule rather than a fly-bye comment, want to add one or create an issue for followup?

Reply:
Agree lets prepare a separate PR for it.